kola/tests/misc/network.go

```diff
@@ -47,13 +49,17 @@ func init() {
 		Name:        "cl.network.listeners.legacy",
 		Distros:     []string{"cl"},
 		EndVersion:  semver.Version{Major: 1967},
+		// This test is normally not related to the cloud environment unless the OEM tools would unexpectedly listen on ports
+		Platforms: []string{"qemu", "qemu-unpriv"},
 	})
 	register.Register(&register.Test{
 		Run:              NetworkInitramfsSecondBoot,
 		ClusterSize:      1,
 		Name:             "cl.network.initramfs.second-boot",
 		ExcludePlatforms: []string{"do"},
 		Distros:          []string{"cl"},
+		// This test is normally not related to the cloud environment unless the OEM tools would unexpectedly listen on ports
```

Comment:
Is this test (cl.network.initramfs.second-boot) still about listening on ports?

Reply:
I thought so but it's not, thanks. The test checks that networks is not started in the initramfs on the second boot. I removed the limitation to qemu as it's an integration with the cloud environment in the initramfs that's good to check.